os/arch/arm/src/amebasmart/hardware/amebasmart_memorymap.h

```diff
@@ -83,7 +83,11 @@
 /* Currently there is no non-cache memory region, no need to map for it */
 /* NS code is covered by PSRAM section */
 #define AMEBASMART_MMAP_DEV0_PSECTION   0x40000000  /* 0x40000000 - 0x440FFFFF  65MB NS Memory mapped device */
+#if CONFIG_RTL8730E_BOARD_REVISION >= 6
```

Comment:
is it okay ?

```suggestion
#if CONFIG_RTL8730E_BOARD_REVISION >= 7
```

Reply:
It depends on what is the HW condition of the version. If revision 6 is DDR it is fine.
I will standardize it to revision 7.

Reply:
DDR board revision is 7. 
revision 6 is audio AMP changes and TSP pin changes